lib/dynamodb_framework/dynamodb_repository.rb

```diff
@@ -187,7 +187,7 @@ def query(partition_key_name, partition_key_value, range_key_name = nil, range_k
     private
 
     def to_hash(obj)
-      HashKit::Helper.new.to_hash(obj)
+      HashHelper.new.to_hash(obj)
```

Comment:
could stick the HashHelper into an instance variable and create if null on to_hash method otherwise use instance variable. Just to prevent this from having to instantiate a new HashHelper object every time the to_hash method is called. thanks